robot/basestation/static/js/elements/feed/camera.js

```diff
@@ -112,6 +112,39 @@ $(document).ready(() => {
     })
   })
 
+  $('#record-video-toggle').on('click', function (event) {
+    var el = $(this); // variable of button text
+
+    // Swap button text out for 'Start recording feed' if button says 'Stop recording feed'
+    // run stop recording feed function in app.py
+    if (el.text() == el.data("text-swap")) {
```

Comment:
This may be a bit more advanced, but I foresee the need to have other "toggle-able" buttons. So having this logic in every of them would not be optimal. Instead there are elegant solutions in this post https://stackoverflow.com/questions/2155453/jquery-toggle-text